src/SmartComponents/Inventory/applications/compliance/SystemPolicyCard.js

```diff
@@ -11,10 +12,16 @@ import {
 import PropTypes from 'prop-types';
 
 class SystemPolicyCard extends React.Component {
-    complianceIcon = () => {
-        const { policy: { compliant }} = this.props;
+    constructor(policy) {
+        super();
+        this.policy = policy.policy;
```

Comment:
Please do not assign to class attributes, but use state. With this approach our component will not be re-rendered once policy is changed.